checkstyle-tester/README.md

```diff
@@ -93,73 +93,70 @@ You can generate report in different ways:
 1) generate report yourself: [manual generation](./README_MANUAL_EXECUTION.md#executing-diffgroovy)
 2) generate report using github action.
 
-To generate report using github action, you need to provide links to "raw" versions
-of your
-configuration([template](https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/my_check.xml))
-and
-projects-to-test-on.properties([template](https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/projects-to-test-on.properties))
-files.
-There are many ways to do that, but recommended one is to create
-[GitHub Gist](https://gist.github.com/) for each file.
-
-After gist is created, click on "Raw" button on right side.
-![Alt text](./screenshots/gist_raw_button.png?raw=true "Gist raw button example")
-
-And copy the link.
-![Alt text](./screenshots/gist_link.png?raw=true "Link example")
-
-Alternative way is to commit file to some repository and branch, get its raw version and copy link.
-![Alt text](./screenshots/github_raw_button.png?raw=true "Repository raw button example")
+To generate report using github action, you need to add specific line(s) to your PR description.
+Please note:
+- Each item should be on new line.
+- There should be at least one config (items 1-3).
+- Placeholders should be replaced with links to ["raw" versions of files](./README_GET_RAW_LINK.md).
+
+Lines to add:
```

Comment:
unfortunately this section is hard to read and it is very scary for new contributor, our main goal is to not scary away new contributor.

lets keep this section as it is good for frequent contributors that need more options.

but what if we make at this level section like:
- Difference Report when you change existing Check
- Difference Report when you change existing Check (with custom projects list and label)
- Difference Report when you create new Check (with different base and patch config, with custom projects list and label)

so we will users exact link to use cases, without bothering to read whole manual and yes we can give them a link to manual section if still not clear.

Reply:
there are too many cases with every option.
I can add some section to the top with name like "Quick start" and several most common options - new check, bugfix and new/different properties in checks without mention of labels or custom projects. These options can be listed after quick start section